src/model/job-advert.v1.yaml

```diff
@@ -0,0 +1,45 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Job advert
+type: object
+allOf:
+  - $ref: ../snippets/job-advert.v1.yaml
+  - properties:
+        description:
+            description: Description
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        responsibilities:
```

Comment:
If we ever have a use for it to be separate, we can extract it(/any others). But for now I don't see a reason to treat it differently.